src/domain/alerts/alerts.repository.ts

```diff
@@ -79,4 +90,28 @@ export class AlertsRepository implements IAlertsRepository {
       }
     }
   }
+
+  private async notifyInvalidTransaction(
+    chainId: string,
+    log: AlertLog,
+  ): Promise<void> {
+    const emails = await this.emailRepository.getVerifiedEmailsBySafeAddress({
+      chainId,
+      safeAddress: log.address,
+    });
+
+    if (emails.length) {
+      return this.emailApi.createMessage({
+        to: emails,
+        template: this.configurationService.getOrThrow<string>(
+          'email.templates.unknownRecoveryTx',
+        ),
+        subject: 'Unknown transaction attempt',
+        substitutions: {},
```

Comment:
The contents for these fields are out of the scope of this PR, and will be reviewed later when real subjects and the needed substitutions in the final template are clarified.

Reply:
I'd add a `TODO` here to keep track that this needs to be done.